src/catalog.js

```diff
@@ -237,7 +239,14 @@ class Page {
   }
 
   static create({ category, parent, slug, sources }) {
-    const matter = frontMatter(sources);
+    var matter;
+
+    try {
+      matter = frontMatter(sources);
+    } catch(error){
+      console.error(error.name + ": Check the YAML formatting of the file: " + chalk.cyan(category + "/" + slug + ":" + error.mark.line) + ", by using a linter.");
```

Comment:
I think to you need to add parent as well if its not null

Reply:
The parent is undefined when you try to. console log it.

Reply:
not alwasy, you need to check if its undefined. its only not defined when the page is in a subfolder.